modules/end-user-guide/partials/proc_configuring-github-oauth.adoc

```diff
@@ -8,21 +8,49 @@
 
 OAuth for GitHub allows for automatic SSH key upload to GitHub.
 
+.Prerequisites
+
+* A running instance of {prod}. To install an instance of {prod}, see {link-installing-an-instance}.
+* The `{orch-cli}` tool is available.
+
 .Procedure
 
-* Set up the link:https://developer.github.com/apps/building-oauth-apps/creating-an-oauth-app[GitHub OAuth client]. The *Authorization callback URL* is filled in the next steps.
+ifeval::["{project-context}" == "che"]
+* For {prod-short} deployed in multi-user mode:
```

Comment:
The configuration of the GitHub OAuth application is common to both multi-user and single-user mode. This line should be moved below.